lookbook/docs/components/side-panel.md.erb

```diff
@@ -0,0 +1,55 @@
+The <strong>Side panel</strong> is right-side pane that is used to show information relevant to the current page. This information normally concerns the contents of the entire page and can generally directly be modified or adjusted via the side panel.
+
+A good example is a meeting occurrence; the side panel is used to show key meeting details (time, location, duration...), status, a list of participants and attachments.
+
+## Anatomy
+
+The side panel consists of a series of sections (or just one section). Each section has a fixed format and contains:
+
+- A section header
+  - with counter (optional)
+- An action button or more icon (optional)
+- Content slot
+- An action link (optional)
+- A divider at the end
+
+These sections are stacked vertically.
+
+The content slot can take a number of different types of content and is relatively free. However, the content has to make sense in visual and semantic hierarchy. It's also best to make sure all objects are aligned to the left edge of the panel.
+
+The content of the side panel should always be secondary to the main content area.
+
+## Best practices
+
+**Do**
+
+- Do use the side panel for contextual information on the page
+- Ensure that elements on a side panel respect the hierarchy of the page and elements in other side panels (the content in each side panel should roughly be at the same hierarchical and visual level)
+- Use `color-fg-subtle` if the text is meant to be helpful guideline text or a caption; use `color-fg-default` for key information.
+
+**Don't**
+
+- Do not use it as a way to create columns
+- Do not make a certain section too long, compared to others. If necessary, use a 'Show more' link to expand it (eg. Participant list in a meeting occurence )
+
+## Used in
+
+Some pages where the side panel is used:
+
+- Meeting occurence (displays meeting details, meeting status, participants and attachments)
+- Project overview (displays project attributes)
+- Project list
+- Some admin index pages (eg. SAML or OpenID provider details page; file storage details page)
+
+## Mobile considerations
+
+By default, side panel move to the bottom of the main content area on mobile. However, there are other considerations to keep in mind:
+
+- Some sections have to manually be moved to the top if they contain critical information (eg. the meeting details for a meeting occurence go to the top, but the participant list goes to the bottom)
```

Comment:
This is not true and technically also not possible. Either everything goes to the top or everything to the bottom, we can't split it (and we don't do it for Meetings)